app/resources/locale/en/en.po

```diff
@@ -1107,134 +1098,132 @@ msgid ""
 "This release has been possible thanks to the great work \n"
 "              done by an emerging team from"
 msgstr ""
+"This release has been possible thanks to the great work \n"
+"              done by an emerging team from"
 
-#: app/scripts/services/graph.js:521
+#: app/scripts/services/graph.js:532
 msgid ""
 "To enter on \"edit mode\" of deeper block, you need to finish current \"edit "
 "mode\", lock the keylock to do it."
 msgstr ""
 "To enter on \"edit mode\" of deeper block, you need to finish current \"edit "
 "mode\", lock the keylock to do so."
 
-#: app/scripts/controllers/design.js:51
+#: app/scripts/controllers/design.js:50
 msgid "To navigate through design, you need to close \"edit mode\"."
 msgstr "To navigate through a design, you need to close \"edit mode\"."
 
 #: app/scripts/controllers/menu.js:349
-#, fuzzy
 msgid ""
-"To save your design you need to lock the keylock and               go to top "
+"To save your design you need to lock the padlock and               go to top "
 "level design.<br/><br/>If you want to export               this submodule to "
 "a file, execute \"Save as\" command to do it."
 msgstr ""
-"To save your design you need to lock the keylock and go to top level design."
-"<br/><br/>If you want to export this submodule to a file, execute \"Save as"
-"\" command."
+"To save your design you need to lock the padlock and               go to top "
+"level design.<br/><br/>If you want to export               this submodule to "
+"a file, execute \"Save as\" command to do it."
```

Comment:
Suggest:
  ('top-level' seems to be the convention)
"To save your design you need to lock the padlock and go back to top-level "
"design.<br/><br/>If you want to export               this submodule to "
"a file, use \"Save as\" command to do it."

Reply:
👌
One request here: I don't edit the *.po file but the source file where the message is located. It would help (be quicker) if you, as you did in many others before, provided also the previous lines of the *.po file where the source code file and line appear (so I don't have to search them every time). Thanks!